awx/main/scheduler/task_manager.py

```diff
@@ -426,18 +403,21 @@ def should_update_related_project(self, job, latest_project_update):
             return True
         return False
 
-    def generate_dependencies(self, task):
-        dependencies = []
-        if type(task) is Job:
+    def generate_dependencies(self, pending_tasks):
```

Comment:
might be better to call the argument something else, like `undeped_tasks`

Reply:
changed variable name to `undeped_tasks`